Feature/filter upstream urls

mrafieee opened the pull request.

Commit:
upstream_url filtered out for skipping bad upstream urls

Commit:
test

calvinlfer reviewed (changes requested):
I'm good with this, but I would like to see something in the README.md about this feature 😄

Commit:
readme added

Commit:
Allow users to specify minimal API backup configuration writing JSON like
```json
        {
          "uris": [
            "/v2/external/bliki"
          ],
          "upstream_url": "https://blikipedia.org/calvin",
          "name": "calvin-bliki"
        }
```
